modules/test/playwright/tests/analytics-settings-web/utils/analyticsSettings.ts

```diff
@@ -142,7 +150,7 @@ export async function syncSite(page, propertyName) {
 
 	await page.getByRole('tab', {name: 'Sites'}).click();
 
-	await page.waitForTimeout(3000);
+	await page.waitForSelector('div[aria-modal="true"] tbody');
```

Comment:
Can we add a specific selector for that modal to avoid selecting the wrong one?

Reply:
Hey @interaminense, this selector will only check if the table with the list of properties is appearing so that the test knows that it can proceed to the next step. Only this modal will appear in this part, so I don't think there's any way it can get the wrong one. What do you think?